kubernetes_env/benchmark/benchmark.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+import argparse
+import logging
+import subprocess
+import os
+import sys
+import signal
+import requests
+import time
+import matplotlib.pyplot as plt
+import json
+from pathlib import Path
+
+sys.path.append("..")
+
+import kube_env
+import kube_util as util
+
+log = logging.getLogger(__name__)
+FILE_DIR = os.path.abspath(os.path.dirname(__file__))
+# Haven't found a workaround
+FORTIO_DIR = Path(__file__).parent.resolve().parent.resolve().parent.resolve()
+
+def plot(xaxis_data, yaxis_data, xaxis_label, yaxis_label, query, graph_name):
+    plt.plot(xaxis_data, yaxis_data, marker='o')
+    plt.xlabel(xaxis_label)
+    plt.ylabel(yaxis_label)
+    plt.title(query)
+    if not check_dir("graphs"):
+      os.mkdir(os.path.join(FILE_DIR, "graphs"))
+    plt.savefig(os.path.join(FILE_DIR, f"graphs/{graph_name}.png"))
+
+def check_dir(dirname):
+    return os.path.isdir(os.path.join(FILE_DIR, dirname))
+
+def make_dir(dirname):
+    os.mkdir(os.path.join(FILE_DIR, dirname))
+
+def run_fortio(platform, threads, qps, run_time):
+    if not check_dir("data"):
+      make_dir("data")
+    _, _, gateway_url = kube_env.get_gateway_info(platform)
+    cmd = f"{FORTIO_DIR}/bin/fortio "
+    cmd += f"load -c {threads} -qps {qps} -jitter -t {run_time}s -json data/output.json "
+    cmd += f"http://{gateway_url}/productpage"
+    fortio_proc = util.start_process(cmd, preexec_fn=os.setsid)
+    outs, errs = fortio_proc.communicate()
```

Comment:
You can probably take the exec_process from utils with the appropriate redirection of stderr and stdout.